### PR TITLE
nuget 5.5.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/nuget.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nuget < Formula
   desc "Package manager for Microsoft development platform including .NET"
   homepage "https://www.nuget.org/"
-  url "https://dist.nuget.org/win-x86-commandline/v5.4.0/nuget.exe" # make sure libexec.install below matches case
-  sha256 "880f8d306a65932b11f7edd7768b57d20e78fc618b21d785b303da27facc9a70"
+  url "https://dist.nuget.org/win-x86-commandline/v5.5.1/nuget.exe" # make sure libexec.install below matches case
```
Comment (Bo98) [inline Formula/nuget.rb:4]: https://www.nuget.org/downloads lists `v5.4.0` as the recommended latest.

If you like JSON then you can also check https://dist.nuget.org/tools.json for `ReleasedAndBlessed`.

Comment (chenrui333) [reply Formula/nuget.rb:4]: Sounds good, I will update livecheck to reflect this information source. 👍

Comment (chenrui333) [reply Formula/nuget.rb:4]: I have to say, I really like `ReleasedAndBlessed` 😄

**Comment by Bo98** [comment]
It's now blessed.

**Comment by chenrui333** [comment]
Nice, 😄 🚀

**Pull request merged**